### DISCUSSION TITLE
Persistent Component for Video Streaming/Call

### DISCUSSION THREAD

**Discussion by annac-117** [vercel/next.js#50893] [Help]
### Summary

I want to add a video streaming component to my app. This component will be accessible once i log in (next-auth + third party authentication provider).

At login, this video streaming component will fetch the required data once and after that whenever I visit any other page or do anything else in those pages, i want that video streaming component to stay there at the right side and not re-render or get refreshed because rendering of that component will interrupt the video streaming.

What is/are the best ways to achieve it?

PS. The video streaming component is connected to redux and has its own internal states.

Also, I'm using the old pages folder structure and not the new app folder structure for my Next.js 13 application.

**Comment by debjits1** [comment]
You can create a placeholder component and keep that visible until the actual component is loaded with the stream
